rotary_utils/repair.py

```diff
@@ -833,63 +803,59 @@ def run_end_repair(long_read_filepath: str, assembly_fasta_filepath: str, assemb
                 f'A summary of repair work is saved at {end_repair_status_filepath}.')
 
 
-def parse_cli(subparsers=None):
+def subparse_cli(subparsers, parent_parser: argparse.ArgumentParser = None):
     """
-    Parses the CLI arguments
```

Comment:
@jmtsuji So, I would try to be consistent in your doc string comments. Either put a blank line between the param and the description sentence or not. I generally put a blank line. Example:

```python
"""
Parses the CLI arguments
:param subparsers An special action object created by argparse's add_subparsers() method.
"""
```

versus 

```python
"""
Parses the CLI arguments

:param subparsers An special action object created by argparse's add_subparsers() method.
"""
```

Reply:
@jmtsuji This may be an issue to put on a different branch.

Reply:
Resolved - added a blank line underneath all function descriptions.